public/conda.json

```diff
@@ -1,16 +1,33 @@
 {
+  "alveoleye": null,
```

Comment:
Do we know why these are no longer sorted into the rest of the file? Was that by design?